docker/build-database.sh

```diff
@@ -52,16 +51,16 @@ mkdir -p "${PGDATA}" && chown -R postgres:postgres "${PGDATA}" && chmod 777 "${P
 # create db-data folder for the repo
 mkdir -p "${DBDATA_PATH}"
 
-# configure repo with the readonly SFTP password given as argument
-echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
+# configure repo with the readonly SFTP password given as argument if set
+[[ -n "${SFTP_READONLY_PASSWORD}" ]] && echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
```

Comment:
```suggestion
[[ -n "${SFTP_READONLY_PASSWORD}" ]] && echo "\$sftp_data_readonly_password = '${SFTP_READONLY_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
[[ -n "${HTTPS_PASSWORD}" ]] && echo "\$https_data_password = '${HTTPS_PASSWORD}'" >> "${DBCONFIG_PATH}/AeriusSettings.User.rb"
```
Not sure if this the correct way to go if both are present, but the https password should be treated similar to the sftp one.

Reply:
ah yeah, didn't think about adding the HTTPS credentials (yet).. Can do that directly..